### ISSUE TITLE
allow creation of a single automated PR per rep rather than multiple analogous PRs

### ISSUE THREAD

**Issue by vchudnov-g** [googleapis/repo-automation-bots#5257]
In the Python monorepo, we are now getting a few PRs doing the same thing for different sets of published packages (eg, "Update to vX.Y.Z". While this is certainly better than the single PR we were getting for each of the split-repos we used to have, it still surprises us that we have more than one PR. Is this because the automation wants to limit the PR size?

What we'd like to have is the option (via some sort of config?) to **generate as single PR rather than these multiple analogous PRs**. That way, in this example, we only see one PR called "Update to vX.Y.Z", and, while it would even more files, these automated PRs are the ones where we do a light coherence check rather than checking line by line anyway, since we trust the automation and rely on tests to flag errors we can investigate carefully.

Doing this would reduce maintenance toil, which is one of our major themes in the Python team.

**Comment by SurferJeffAtGoogle** [comment]
Yes, the PR size is [limited to 20](https://github.com/googleapis/repo-automation-bots/blob/main/packages/owl-bot/src/bin/commands/scan-googleapis-gen-and-create-pull-requests.ts#L95).

It was requested in [issue 4763](https://github.com/googleapis/repo-automation-bots/issues/4763).

This will probably take about 2 hours to implement.  @vchudnov-g, please confirm this will save you more than 2 hours over the next 2 years if implemented.
